src/uu/stty/src/flags.rs

```diff
@@ -313,3 +317,41 @@ pub const BAUD_RATES: &[(&str, BaudRate)] = &[
     ))]
     ("4000000", BaudRate::B4000000),
 ];
+/// Control characters for the stty command.
+///
+/// This constant provides a mapping between the names of control characters
```

Comment:
needs a rustfmt

Reply:
Oops, forgot to `git add`... should be there now